.github/workflows/hotfix-wf-service.yml

```
@@ -0,0 +1,377 @@
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          submodules: 'recursive'
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+
+      - name: Cache Docker layers
+        id: cache
+        uses: actions/cache@v2
```

actionlint

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)